src/database.ts

```diff
@@ -55,12 +63,22 @@ function initializeModel(
 export const Database = {
     ROLES: initializeModel("roles", RoleSchema, RoleValidator),
     EVENTS: initializeModel("events", EventSchema, privateEventValidator),
+    EVENTS_ATT: initializeModel(
```

Comment:
rename the EVENTS_ATT to be more specific, same w ATTENDEE_ATT

Reply:
cool EVENTS_ATTENDANCE and ATTENDEE_ATTENDANCE good?